src/libraries/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -528,6 +528,55 @@ public SequencePosition GetPosition(long offset)
             return Seek(offset);
         }
 
+        /// <summary>
+        /// Translate opaque <see cref="SequencePosition"/> value to numerical position offset
+        /// </summary>
+        /// <param name="position"></param>
```

Comment:
@MarcoRossignoli Thank you for adding the triple slash section. Can you please make sure you fill out the param and the returns? Also please make sure to add any exceptions that this method may throw.

Reply:
Sure sorry.

Reply:
Updated let me know if it's ok.